generators/cmd/templates/root.gotmpl

```diff
@@ -28,6 +29,7 @@ func init() {
   RootCmd.PersistentFlags().StringVar(&providedAPIToken, "api-token", "", "Specify API token otherwise soracom-cli performs authentication on behalf of you")
   RootCmd.PersistentFlags().StringVar(&providedAuthKeyID, "auth-key-id", "", "Specify AuthKeyId to be used for authentication. If both --auth-key-id and --auth-key are specified, soracom-cli works without profiles. It means you don't need to run 'soracom configure' before running the command. This may be suitable for temporary execution. Please don't forget specifying these options along with --coverage-type.")
   RootCmd.PersistentFlags().StringVar(&providedAuthKey, "auth-key", "", "Specify AuthKey to be used for authentication. If both --auth-key-id and --auth-key are specified, soracom-cli works without profiles. It means you don't need to run 'soracom configure' before running the command. This may be suitable for temporary execution. Please don't forget specifying these options along with --coverage-type.")
+  RootCmd.PersistentFlags().StringVar(&providedCommandToProvideCredentials, "command-to-provide-credentials", "", "Specify a command to privide authentication credentials. If this option is specified, soracom-cli works without profiles. It means you don't need to run 'soracom configure' before running the command. This may be suitable for temporary execution. Please don't forget specifying these options along with --coverage-type.")
```

Comment:
The command given here can provide a profile, not just credentials, so it may be better to use a different name. e.g. `--profile-command`

ここで与えられるコマンドはクレデンシャルに限らずプロファイルを提供することができますので、違う名前にしたほうが良いかもしれません。たとえば `--profile-command` 等。

Reply:
@bearmini  I modified `CommandToProvideCredentials` to `ProfileCommand`.